web/scm/src/app/app-component/app.component.scss

```diff
@@ -0,0 +1,34 @@
+.app-container {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+}
+.scm-toolbar {
+    background: #fff;
+    box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.12), inset 0 -1px 0 0 #dadce0;
+    left: 0;
+    right: 0;
+    top: 0;
+    z-index: 2;
+}
+.product-name-container {
+    display: flex;
+    align-items: center;
+}
+.product-name {
+    color: #5f6368;
```

Comment:
Is it possible to make it part of theme or move to main styles? Imagine we will need to have dark theme, it would be nice to have all colors in one place